gatling-core-java/src/main/java/io/gatling/javaapi/core/Session.java

```diff
@@ -43,6 +44,7 @@ public Session(io.gatling.core.session.Session wrapped) {
    * @param <T> the type of the desired value
    * @return the value if it exists, null otherwise
    */
+  @Nullable
```

Comment:
For consistency:
```java
public <T> @Nullable T get(@Nonnull String key)
```

Reply:
Interesting. That was what I wrote in the first place, but IntelliJ warned "'Nullable' not applicable to type use".
It compiles fine apparently.